src/types/rest/response/public-data.ts

```diff
@@ -149,3 +149,33 @@ export interface FundingRateHistory {
 export interface SystemTime {
   ts: string;
 }
+
+export interface OptionsTradeInfo {
+  instId: string;
+  tradeId: string;
+  px: string;
+  sz: string;
+  side: 'buy' | 'sell';
+  ts: string;
+}
+
+export interface OptionTrade {
+  vol24h: string;
+  optType: 'C' | 'P';
+  tradeInfo: OptionsTradeInfo[];
+}
+
+export interface GetOptionTrades {
+  instId: string;
+  instFamily: string;
+  tradeId: string;
+  px: string;
+  sz: string;
+  side: 'buy' | 'sell';
+  optType: 'C' | 'P';
+  fillVol: string;
+  fwdPx: string;
+  idxPx: string;
+  markPx: string;
+  ts: string;
+}
```

Comment:
This should be on the request side:
https://github.com/tiagosiebler/okx-api/blob/master/src/types/rest/request/public.ts

Would also rename to `OptionTradesRequest` for consistency

Reply:
There is request for option trades, this is response. I will rename it so its more clear.

Reply:
Checked the docs, yep Im right. So changed the name. All requests always have suffix request, so if you see that somewhere let me know.